docs/creature_template.md

```diff
@@ -577,7 +557,21 @@ Used to modify the base Level/Class armor of a creature.
 
 #### DamageModifier
 
-TODO!
+Used to modify the Minimum/Maximum damage of a creature.
```

Comment:
useless comment tbh.
it doesn't matter what it did in the past, what matters is what it does now

Reply:
Maybe it's more of a grammar mistake. I mean it like "(This value is) used to...". Will try to make it a bit easier to understand.

Reply:
"This field is ued to..." could work

Reply:
Should I change it for the other Modifiers too? Thats where I got the sentence from.

Reply:
Actually you can leave it as-is